packages/kbn-plugin-helpers/src/cli.ts

```diff
@@ -93,5 +95,73 @@ export function runCli() {
         }
       },
     })
+    .command({
+      name: 'dev',
+      description: `
+        Builds the current plugin ui browser side so it can be picked up by Kibana
+        during development
+
+      `,
+      flags: {
+        boolean: ['dist', 'watch'],
+        alias: {
+          d: 'dist',
+          w: 'watch',
+        },
+        help: `
+          --dist, -d  Outputs bundles in dist mode instead
+          --watch, -w  Starts the watch mode
+        `,
+      },
+      async run({ log, flags }) {
+        const dist = flags.dist;
+        if (dist !== undefined && typeof dist !== 'boolean') {
+          throw createFlagError('expected a single --dist flag');
+        }
+
+        const watch = flags.watch;
+        if (watch !== undefined && typeof watch !== 'boolean') {
+          throw createFlagError('expected a single --watch flag');
+        }
+
+        const found = findPluginDir();
+        if (!found) {
+          throw createFailError(
+            `Unable to find Kibana Platform plugin in [${process.cwd()}] or any of its parent directories. Has it been migrated properly? Does it have a kibana.json file?`
+          );
+        }
+
+        if (found.type === 'package') {
```

Comment:
With this check we can't use `example` plugins via the CLI anymore - is this intended?